docs/src/whatsnew/3.1.rst

```diff
@@ -257,11 +312,44 @@ This document explains the changes made to Iris for this release
 #. `@pp-mo`_ reworked the netcdf loading code, replacing Pyke rules with a pure Python implementation.
    (:pull: `4198`)
 
-
 #. `@lbdreyer`_ updated the CF standard name table to the latest version: 
    `v77 <http://cfconventions.org/Data/cf-standard-names/77/src/cf-standard-name-table.xml>`_.
    (:pull:`4282`)
 
+=========================================
+Internal changes in v3.0.4 (22 July 2021)
+=========================================
+#. `@bjlittle`_ Unpinned the `cftime`_ package dependency within Iris in order
+   to allow use of the latest versions of `cftime`_, `cf-units`_ and `nc-time-axis`_.
+   (:pull:`4222`)
+
+#. `@rcomer`_ modified test modules so they run consistently under ``pytest`` and
+   ``nose``, and also fixed some minor issues with :class:`~iris.time.PartialDateTime`.
+   (:pull:`4249`)
+
+========================================
+Internal changes in v3.0.2 (27 May 2021)
+========================================
+
+#. `@jamesp`_ updated a test to the latest numpy version (:pull:`3977`) [``pre-v3.1.0``]
```

Comment:
should these still have the `pre-v3.1.0` ?

Reply:
Maybe include the note
```
   Note that, the above contributions labelled with ``pre-v3.1.0`` are part of the forthcoming
   Iris v3.1.0 release, but require to be included in this patch release.
```

Reply:
Good spot !
I hadn't properly understood what these were there for.

Reply:
I think it means that these entries should be in the 'main' release part, not the bugfix sections.
Doing that now.